src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -210,6 +208,20 @@ private FulfilmentRequestedEvent createFulfilmentEvent(
         Region.valueOf(caseServiceClient.getCaseById(caseId, false).getRegion().substring(0, 1));
     Product product = findProduct(fulfilmentCode, deliveryChannel, region);
 
+    boolean requestIsForIndividualProduct =
+        product.getCaseType() == Product.CaseType.HI
```

Comment:
better : if (!isHouseholdOrCommunal)

Reply:
Now using the local method for deciding if household or communal